examples/resources/plugins/teleport-slack.toml

```diff
@@ -1,19 +1,20 @@
 # example slack plugin configuration TOML file
 [teleport]
-auth-server = "example.com:3025"  # Teleport Auth Server GRPC API address
-client-key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key
-client-crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate
-root-cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs
+auth_server = "example.com:3025"                        # Teleport Auth Server GRPC API address
+client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key
+client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate
+root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs
 
 [slack]
-token = "api-token"       # Slack Bot OAuth token
-secret = "signing-secret-value"   # Slack API Signing Secret
-channel = "channel-name"  # Slack Channel name to post requests to
+token = "api_token"             # Slack Bot OAuth token
+secret = "signing-secret-value" # Slack API Signing Secret
+channel = "channel-name"        # Slack Channel name to post requests to
 
 [http]
-listen = ":8081"          # Slack interaction callback listener
-# https-key-file = "/var/lib/teleport/plugins/slack/server.key"  # TLS private key
-# https-cert-file = "/var/lib/teleport/plugins/slack/server.crt" # TLS certificate
+# listen_addr = ":8081" # Network address in format [addr]:port on which callback server listens, e.g. 0.0.0.0:443
```

Comment:
Port at the end should match the port at the beginning

i.e. `# Network address in format [addr]:port on which callback server listens, e.g. 0.0.0.0:8081`